technical-vision.md

```diff
@@ -1,9 +1,28 @@
 # OpenSSF Technical Vision
 
-We envision a future where participants in the open source ecosystem use and share high quality software, with security handled proactively, by default, and as a matter of course:
-- Developers can easily learn secure development practices and are proactively guided by their tools to apply those practices and automatically informed when action is needed to prevent, remediate, or mitigate security issues.
-- Developers, auditors, and regulators can create and easily distribute security policies that are enforced through tooling and automation, providing continuous assurance of the results.
-- Developers and researchers can identify security issues (including unintentional vulnerabilities and malicious software) and have this information swiftly flow backwards through the supply chain to someone who can rapidly address the issue.
-- Community members can provide information and notifications about product defects, mitigations, quality, and supportability and have this information rapidly flow forward across the ecosystem system to all users, and users can rapidly update their software or implement mitigations as appropriate.
+An aspirational view of the future we seek to create in 2-5 years:
 
-The Open Source Security Foundation (OpenSSF) provides tools, services, training, infrastructure, and resources to achieve this vision.
+### OpenSSF is a well-respected strategic partner to large OSS foundations and individual OSS projects
+- OpenSSF is clearly positioned within broader ecosystem with a clear, altruistic charter
+- Periodically review the value proposition of OpenSSF with key stakeholders
```

Comment:
I may have missed this in an earlier draft, but who are the key stakeholders? OpenSSF membership? The governing board? The open source ecosystem? All of the above?

Reply:
#165

Reply:
How often should this periodic review take place? This doesn't necessarily have to be answered here but we ought to have that documented somewhere or I'm afraid it won't get implemented.

Reply:
The foundation publishes an annual report, but my 2c: the frequency of meetings needs to be quarterly, perhaps with a more frequent newsletter to drive awareness of all the good things going on within the community.

Reply:
I think, depending on which stakeholder, quarterly could be achievable (i.e. - checking in with the GB).  I don't know we have the resources or capabilities to, for example, talk to "oss consumers" with that high a frequency without setting up surveys of townhalls/office hours/etc.  In TAC PR165(https://github.com/ossf/tac/issues/165) I put my initial thoughts on our key stakeholder groups.  My thoughts and conversation welcome, and once we land on the groups, we could fine-tune cadence/channels to engage them.

Reply:
> The foundation publishes an annual report, but my 2c: the frequency of meetings needs to be quarterly, perhaps with a more frequent newsletter to drive awareness of all the good things going on within the community.

I also like the newsletter idea, summarizing activities and upcoming items.

Reply:
> The foundation publishes an annual report, but my 2c: the frequency of meetings needs to be quarterly, perhaps with a more frequent newsletter to drive awareness of all the good things going on within the community.

Sorry, which meetings are you referring to?

Reply:
The meetings to review the value prop with stakeholders referred to by this bullet, but those meetings are just an idea at this point.